lib/templates.js

```diff
@@ -84,29 +115,40 @@ module.exports = function(client) {
       };
       return client.delete(options).asCallback(callback);
     },
-    preview: function(options, callback) {
+    /**
+     * Preview the most recent version of an existing template by id.
+     *
+     * @param {string} id
+     * @param {Object} options
+     * @param {RequestCb} [callback]
+     * @returns {Promise}
+     */
+    preview: function(id, options, callback) {
       var reqOpts;
       options = options || {};
 
-      if(!options.id) {
+      // Handle optional options argument
+      if (typeof options === 'function') {
```

Comment:
We're basically making a contract that you can do `(<int>, <object>, <function>)` or `(<int>, <function>)` and in both cases the function is optional. If you do `(<int>, <function>, <function>)` too bad for you :P